src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerBuildImage.groovy

```diff
@@ -95,9 +104,13 @@ class DockerBuildImage extends AbstractDockerRemoteApiTask implements RegistryCr
             buildImageCmd = dockerClient.buildImageCmd(getInputDir())
         }
 
-        if (getTag()) {
+        if(getTag()) {
             logger.quiet "Using tag '${getTag()}' for image."
             buildImageCmd.withTag(getTag())
+        } else if (getTags()) {
+            def tagListString = getTags().collect {"'${it}'"}.join(", ")
+            logger.quiet "Using tags ${tagListString} for image."
+            buildImageCmd.withTags(getTags())
```

Comment:
Can we allow for both? Maybe something like:

```
def allTags = [] + getTags()
if (getTag()) { allTags << getTag() }
if (allTags) {
    logger.quiet "Using tags ${tagListString} for image."
    buildImageCmd.withTags(allTags)
}
```
May be a better more succinct way to write this but I can def see folks defining a default tag and then possibly through a `doFirst` block add additional tags at runtime.